data_reports/119_OakRidgeNationalLab_TDE.Rmd

```diff
@@ -1,30 +1,194 @@
----
-title: "119_OakRidgeNationalLab_TDE"
-author: "Julie Bielecki"
-date: "3/28/2021"
-output: html_document
----
+#Oak Ridge National Lab_TDE
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+The Oak Ridge National Lab_TDE data set in ISCN3 contains 1176 layer-level rows and 294 profile-level information rows after cleaning for ISCN3.5.
+Rows weren't removed from ISCN3 because of reasons.
+Rows weren't removed from ISCN3 because there were no duplicated entries within the layer-level and profile-level information.
+Columns were removed from ISCN3 where information was not available throughout the data set.
```

Comment:
Can you clarify what you mean here? It sounds like you removed columns with partial information instead of removing columns that were made up entirely of `NA`'s

Reply:
Fixed!